ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -73,9 +82,16 @@ int
 azure_connection::put_object(const std::string &object_key, const std::string &file_path) const
 {
     auto blob_client = _azure_client.GetBlockBlobClient(_object_prefix + object_key);
-    // UploadFrom will always return a UploadBlockBlobFromResult describing the state of the updated
-    // block blob so there's no need to check for errors.
-    blob_client.UploadFrom(file_path);
+    // UploadFrom will return a UploadBlockBlobFromResult describing the state of the updated
```

Comment:
Can we update this comment slightly?
```
// UploadFrom returns a UploadBlockBlobFromResult object describing the state of the updated
// block blob on success and throws an exception on failure.
```

Reply:
resolved here https://github.com/wiredtiger/wiredtiger/pull/8750/commits/e42f214727eda92b86bcea0d45fc6c00676173e8

Reply:
Thanks for the suggestions @tammybailey I've taken them onboard